### PR TITLE
src/pages: add ProfilePage + ProfileTabs component for displaying user profile content

### PR DESCRIPTION
**Pull request by maceksimon**
Add Profile page:
* Specify profile routes (with children corresponding to navigation tabs)
* Add `ProfilePage` page layout
* Add `ProfileTabs` component to display tabs on ProfilePage
* Add translations
* Add test for `ProfileTabs`

### CODE REVIEW

File: src/components/profile/ProfileTabs.vue [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        :label="$t('profile.tabNotifications')"
+        data-cy="profile-tabs-button-notifications"
+      />
+    </q-tabs>
+    <!-- Separator -->
+    <q-separator />
+    <!-- Tab panels -->
+    <q-tab-panels v-model="activeTab" animated>
+      <!-- Panel: Details -->
+      <q-tab-panel name="details" data-cy="profile-tabs-panel-details">
```
Comment (tmszi) [inline src/components/profile/ProfileTabs.vue:79]: `name` prop value should be TS Enum.

File: src/components/profile/ProfileTabs.vue [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      v-model="activeTab"
+      class="text-grey"
+      active-color="primary"
+      indicator-color="primary"
+      align="center"
+      data-cy="profile-tabs"
+    >
+      <q-route-tab
+        :to="routesConf['profile_details'].path"
+        name="details"
```
Comment (tmszi) [inline src/components/profile/ProfileTabs.vue:51]: `name` prop value should be TS Enum.

File: src/components/profile/ProfileTabs.vue [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        data-cy="profile-tabs-button-notifications"
+      />
+    </q-tabs>
+    <!-- Separator -->
+    <q-separator />
+    <!-- Tab panels -->
+    <q-tab-panels v-model="activeTab" animated>
+      <!-- Panel: Details -->
+      <q-tab-panel name="details" data-cy="profile-tabs-panel-details">
+        Personal details
```
[the remark this answers is not in the document]
Comment (maceksimon) [reply src/components/profile/ProfileTabs.vue]: Updated. Used comments to indicate placeholder content.

File: src/components/profile/ProfileTabs.vue (hunk 2 above)
Comment (maceksimon) [reply src/components/profile/ProfileTabs.vue:58]: Used TS Enum.

File: src/components/profile/ProfileTabs.vue (hunk 1 above)
Comment (maceksimon) [reply src/components/profile/ProfileTabs.vue]: Used TS enum for tab names.

File: src/components/profile/ProfileTabs.vue (hunk 2 above)
Comment (tmszi) [reply src/components/profile/ProfileTabs.vue:58]: You still use string literal `name="details"`.

Comment (maceksimon) [reply src/components/profile/ProfileTabs.vue]: Sorry, misread the correct point in the code. Fixed now.

**Review by tmszi — APPROVED** [review]

**Pull request merged**